### DISCUSSION TITLE
Data deletion from specific space

### DISCUSSION THREAD

**Discussion by ankitsoin** [finos/TimeBase-CE#285] [Q&A]
Hi @alex-karpovich,

I can find below code which deletes the data based on specific symbols from a stream. **But may I know how to delete the specific symbols data from specific space? using python**

**#Remove data for "APPL" symbol only
identities = ['APPL']
stream.clear(identities)**

Regards,
Ankit

**Comment by alex-karpovich** [comment]
For now we do not have such API methods, please file a feature requests in `issues` tab

**Reply by ankitsoin** [reply]
Thanks for your response @alex-karpovich ,
I have raised the issue.

May I know do we have any rough estimates(in days, weeks will be fine) by when we can expect this feature in community version?

**Reply by ankitsoin** [reply]
Hi @alex-karpovich, may I know, is the feature mentioned in below link available in Enterprise Edition?
https://github.com/finos/TimeBase-CE/issues/286 
Thanks.

**Reply by alex-karpovich** [reply]
if you are partitioning by symbols, then you do not need to provide space names to truncate your data, data will removed automatically.

same, if you are partitioning by date, then for specific symbols data will removed automatically. you just need to provide correct range.

**Comment by ankitsoin** [comment]
Thanks for the response @alex-karpovich 
I can see the delete function in documentation but it is available for java, .net. Do we have this functionality in python as well?

https://kb.timebase.info/docs/technology/apis/tickstream#delete

**Reply by ankitsoin** [reply]
Hi @alex-karpovich, can you please help me with the documentation link where I can provide list of symbols and time-range to delete only provided symbols(list)  in python?

currently we have spaces based on date.
